uktrade/data-hub-frontend
# Reactify investment collection nav

## Description of change

The local nav for switching between the three investment `CollectionLists` has been reactified. The three `CollectionList`s have also been moved into the `modules` directory without being changed further. I would recommend reviewing this PR by commit to hide all the noise caused by this move.

This PR also contains a fix for `project-edit-recipient-spec`

## Test instructions

Go to the investments index page. You should be able to switch between the three `CollectionList` tabs and use them as before.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
